templates/opencraft-nginx.j2

```diff
@@ -0,0 +1,57 @@
+server {
+    listen *:80;
+
+    server_name {{ OPENCRAFT_DOMAIN_NAME }} www.{{ OPENCRAFT_DOMAIN_NAME }};
```

Comment:
@smarnach Not really. I was just pointing out that despite `server_name` being set, this nginx server will respond to every domain.